src/app/pages/orders/components/order-item/order-item.component.html

```diff
@@ -0,0 +1,20 @@
+<span class="text-center text-size-24 font-bold"># {{ order.id }}</span>
+
+<div class="flex gap-5 flex-wrap w-64 justify-center whitespace-break-spaces overflow-ellipsis">
+  <div *ngFor="let product of order.products" class="flex flex-col items-center gap-1">
+    <img [src]="product['image']" class="w-20" alt="Product image">
+
+    <span class="text-center text-blue-600 font-bold">{{ product['productName'] }}</span>
```

Comment:
read the property of the object explicitly

Reply:
I can not access the property of the object explicitly in this situation